addons/prometheus-server/parameter.cue

```diff
@@ -24,9 +27,11 @@ parameter: {
 	// +usage=Specify the service type for expose prometheus server. If empty, it will be not exposed.
 	serviceType: *"ClusterIP" | "NodePort" | "LoadBalancer" | ""
 	// +usage=Specify the storage size to use. If empty, emptyDir will be used. Otherwise pvc will be used.
-	storage: *"" | string
+	storage?: string
```

Comment:
What happens if the storage is `abc`?

Reply:
Kubernetes apiserver will reject the resource creation request.

Reply:
Maybe this is more clear:
```cue
storage: {
   type: *"EmptyDir" | "PVC" 
   size?: int
}
```

Reply:
Do we need to change type from "PVC" to "PersistentVolumeClaim"?